src/Template/Media/get_detail.ctp

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
Rather than creating a new file, I would repurpose the `detail.ctp` file by moving it into the `Media` folder and adding the raw content (minus any CSS) from this file into that one. You'll see that one already has the default layout attached, which will get the menu bar, so all you need are lines 95-end put into the `<body>` section.